static/app/components/group/suggestedOwners/ownershipRules.tsx

```diff
@@ -27,6 +27,26 @@ type Props = {
   project: Project;
 };
 
+const CodeOwnersCTA = HookOrDefault({
+  hookName: 'component:codeowners-cta',
+  defaultComponent: ({organization, project}) => (
+    <SetupButton
+      size="xsmall"
+      priority="primary"
+      to={`/settings/${organization.slug}/projects/${project.slug}/ownership/`}
+      onClick={() =>
+        trackIntegrationAnalytics('integrations.code_owners_cta_setup_clicked', {
```

Comment:
@NisanthanNanthakumar you don't need analytics on the default component tbh, this is only going to be used for self-hosted and development